src/debug-integration.adoc

```diff
@@ -107,3 +110,22 @@ The <<dscratch1c>> register is a CLEN-bit plus tag bit extension to
 
 .Debug scratch 1 capability register
 include::img/dscratch1creg.edn[]
+
+[#dinfc,reftext="dinfc"]
+==== Debug Infinite Capability Register (dinfc)
+
+The <<dinfc>> register is a CLEN-bit plus tag bit CSR only accessible in debug mode.
+
+The reset value is the <<infinite-cap>> capability. The M field (if
+{cheri_mode_ext_name} is implemented) represents the current mode and
+will be only updated if <<MODESW>> is executed.
+
+<<dinfc>> is read/write but with no writeable fields, and so writes are
```

Comment:
Wouldn't it be simpler to declare it is read-only and if needed relaxing that restriction in the future? I'm not sure I see the need for writing even with multiple formats.

Reply:
It can't be read-only because all the debug-only CSRs are also in the read/write CSRs section.

Does make me wonder why they bothered with the RW/RO distinction at all tbh.

Reply:
I think it's more problematic to change teh type of the register and I'm not aware of that ever happening. The difference is trivial really - either you return an error to the debug host on write or you ignore the write. Also the RISC-V debug arch. doesn't include _any_ read only debug mode CSRs, so having it as DRW fits much better without being irregular.

Reply:
Fair enough, happy to keep it RW with all written bits ignored.